docs/MapColonies/Raster/services/export.mdx

```diff
@@ -31,16 +31,11 @@ You can create export tasks via the [Exporter-Trigger service API](/docs/MapColo
 
 You can see the full OpenAPI spec [here](<RASTER-EXPORT-SERVICE_URL>/docs/api)
```

Comment:
You did not update the `OpenAPI` file...

Reply:
there isn't - this should have been directed to the "dev zone" or the service "openAPI"
Note: this format exists in other teams